src/monitoring_service/database.py

```diff
@@ -84,11 +84,12 @@ def get_monitor_request(
         kwargs = {
             key: val
             for key, val in zip(row.keys(), row)
-            if key not in ("non_closing_signer", "saved_at", "waiting_for_channel")
```

Comment:
Was that a bug before?

Reply:
No, we generate the `non_closing_signer` out of the signature when constructing a `MonitorRequest` object. That column is only added to the DB to enforce the correct uniqueness of MRs.

The current handling of `MonitorRequest`, `UnsignedMonitorRequest`, 'non_closing_signer' and `non_closing_participant` is not ideal anymore, but I wanted to keep the changes small for now.

Reply:
Ok, great!